source/PlayState.hx

```diff
@@ -1476,11 +1476,9 @@ class PlayState extends MusicBeatState
 		char.y += char.positionArray[1];
 	}
 
-	public function startVideo(name:String)
+	public function startVideo(name:String, loop:Bool = false, haccelerated:Bool = true, pauseMusic:Bool = false)
```

Comment:
Uh so how would we move the video and other shit? Plus what's `haccelerated` ans `pauseMusic`?

Reply:
bg.x, bg.y, etc

Reply:
will probbly remove loop, haccelered, etc, as those functions are mainly used on actual VLC playbacks rather than on... FNF...